src/Symfony/Bridge/Twig/Resources/views/Form/bulma_0_layout.html.twig

```diff
@@ -0,0 +1,97 @@
+{% extends "form_div_layout.html.twig" %}
+
+{%- block form_widget_compound -%}
+    <div {{ block('widget_container_attributes') }} class="columns is-multiline">
+        {%- if form is rootform -%}
+            {{ form_errors(form) }}
+        {%- endif -%}
+        {{- block('form_rows') -}}
+        {{- form_rest(form) -}}
+    </div>
+{%- endblock form_widget_compound -%}
+
+{% block form_widget_simple -%}
+    {% if errors|length > 0 -%}
+        {% set attr = attr|merge({class: (attr.class|default('') ~ ' is-danger')|trim}) %}
+    {% endif %}
+    {{- parent() -}}
+{%- endblock form_widget_simple %}
+
+{% block form_row -%}
+    <div{% with {attr: row_attr|merge({class: (row_attr.class|default('') ~ ' column is-' ~ column_is|default(6))|trim})} %}{{ block('attributes') }}{% endwith %}>
+        {{- form_label(form) -}}
+        {{- form_widget(form, {attr: {'class': 'input'}}) -}}
+        {{- form_help(form) -}}
+        {{- form_errors(form) -}}
+    </div>
+{%- endblock form_row %}
+
+{% block form_label -%}
+    {% set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' label')|trim}) %}
+    {{- parent() -}}
+{%- endblock form_label %}
+
+{% block textarea_row -%}
+    {% set column_is = 12 %}
+    {{- block('form_row') }}
+{%- endblock textarea_row %}
+
+{%- block textarea_widget -%}
+    {% set attr = attr|merge({class: (attr.class|default('') ~ ' textarea')|trim}) %}
+    {{- parent() -}}
+{%- endblock textarea_widget -%}
+
+{%- block choice_widget_collapsed -%}
+    <div class="select is-fullwidth">
+        {{- parent() -}}
+    </div>
+{%- endblock choice_widget_collapsed -%}
+
+{% block choice_widget_expanded -%}
+    {% set type_class = multiple == true and expanded == true ? 'checkbox' : 'radio' %}
+
+    <div class="control">
+        {% for child_value, child in form %}
+            <label class="{{type_class}}">
+                {{ form_widget(child) }}
+                {% if choice_translation_domain is same as(false) %}
+                    child.vars.label
+                {% else %}
+                    {{child.vars.label|trans({}, choice_translation_domain)}}
+                {% endif %}
+            </label>
+        {% endfor %}
+    </div>
+{%- endblock choice_widget_expanded %}
+
+{%- block file_row -%}
+    {% set onChange = "document.getElementById('" ~ id ~ "_file_name').innerHTML = this.value;" %}
```

Comment:
Don't put inline JS. That would be incompatible with CSP.

Reply:
But if I don't add this JS, when user to select file, this button wont change nothing and he doesn't know if the file was selected or no.

do you have some suggestion to this?

![Screenshot from 2020-05-09 16-56-22](https://user-images.githubusercontent.com/6358755/81483636-0b047a00-9216-11ea-8473-0b829bae12d4.png)

Reply:
Hey @stof I removed JS, could you check again please